client/my-sites/upgrades/checkout-thank-you/jetpack-plan-details.jsx

```diff
@@ -49,13 +55,17 @@ const JetpackPlanDetails = ( { selectedSite } ) => {
 			);
 			props.buttonText = i18n.translate( 'Installation Instructions' );
 			props.href = 'https://en.support.wordpress.com/setting-up-premium-services/';
+			props.onClick = trackManualInstall( 'calypso_plans_autoconfig_click_manual_install' );
```

Comment:
Since I also call trackManualInstall on the other link (a few lines below this), I think I'm going to leave the function as-is.